documentation/modules/exploit/windows/fileformat/cve_2017_8464_lnk_rce.md

```diff
@@ -7,7 +7,7 @@ A fix was released in the June 2017 Patch Tuesday.
 
 ## Vulnerable Setup
 
-To set up the vulnerable environment, install a Windows version without the patch for CVE-2017-8464. To test the bypass, ensure that MS10-046 & MS15-020 are installed.
+To set up the vulnerable environment, install a Windows version without the patch for CVE-2017-8464. To test the bypass, ensure that MS10-046 & MS15-020 are exploitable.
```

Comment:
this sentence doesn't make much sense to me in general. There are no prior mentions in the docs to ms10-046 and a 'bypass' is only mentioned in the module info.  I think more info needs to be added to what we're bypassing, and why ms10 is relevant here.

Reply:
Based on [this](https://www.cvedetails.com/cve/cve-2017-8464) and [this](https://www.rapid7.com/db/vulnerabilities/msft-cve-2017-8464), you aren't bypassing anything. Should the sentence just be removed?

Reply:
https://www.rapid7.com/db/modules/exploit/windows/fileformat/cve_2017_8464_lnk_rce mentions bypassing a CPL (control panel?) Whitelist

Reply:
@zeroSteiner any insight here?

Reply:
Yeah, my understanding was that setting the folder ID in the `SpecialFolderDataBlock` to the Control Panel was causing the white list to be bypassed allowing an arbitrary CPL, (in the case of the module a DLL) to be loaded.

I don't see any reference to MS10-046 in either the fileformat or local variants of the module. IIRC, I didn't write the documentation for either of those so I'm not entirely sure how MS10-046 relates to them.